docs/basic-docs/atomicdex-api-20-dev/enable_z_coin_init.md

```diff
@@ -0,0 +1,80 @@
+<b>To enable Z coins you also need to [install some Zcash Params](https://forum.komodoplatform.com/t/installing-zcash-params/603)</b>
+
+
+#### Arguments
+
+| Structure                                 | Type            | Description                                                                                                                                                          |
+| ----------------------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ticker                                    | string          | Ticker of coin to activate                                                                                                                                           |
+| activation_params                         | object          | Contains details required for activation as explained below                                                                                                          |
+| activation_params.required_confirmations  | integer         | Block confirmations to wait for transactions when doing a swap. Optional, defaults to `3`. Overrides value if set in `coins` file.                                            |
+| activation_params.requires_notarization   | boolean         | For [dPoW](https://komodoplatform.com/en/blog/dpow-demystified/) protected coins, a `true` value will wait for transactions to be notarised when doing swaps. Optional, defaults to `false`. Overrides value if set in `coins` file.  |
+| activation_params.mode.rpc                | string          | Set as `Light` to use external electrum & lightwallet_d servers or `Native` to use local block chain data. If native, the `rpc_data` fields below are not required.  |
+| activation_params.mode.rpc_data           | list of objects | Contains details about servers to be used for `Light` mode operation.                                                                                                |
+| ..rpc_data.electrum_servers.protocol      | string          | Transport protocol used by AtomicDEX API to connect to the electrum server (`TCP` or `SSL`). Optional, defaults to `TCP`                                             |
+| ..rpc_data.electrum_servers.urls          | string          | Urls which are hosting electrum servers                                                                                                                              |
+| ...electrum_servers.disable_cert_verification | boolean     | If `true`, this disables server SSL/TLS certificate verification (e.g. to use self-signed certificate). <b>Use at your own risk</b>                                  |
```

Comment:
We dont have vector of `urls` in `electrum_servers`. 
We have list of `ElectrumRpcRequest` with parameters: `url`(one string), `protocol` and `disable_cert_verification`. `ElectrumRpcRequest` is structure in our code, we dont use its name in curl, we just use arguments of this structure in curly brackets.

If user wants to use one `url` and put `true` in `disable_cert_verification`, list will look like:
`\"electrum_servers\": [{\"url\":\"zombie.sirseven.me:10033\", \"disable_cert_verification\":true}],`.

Several urls in `electrum_servers` will look like:
`\"electrum_servers\": [{\"url\":\"zombie.sirseven.me:10033\"}, {\"url\":\"zombie.sirseven.me:10033\"}],`.

So basically `electrum_servers` has one parameter, which is a list of `{url, protocol, disable_cert_verification}`

@artemii235 am I right?

Reply:
Updated the table, thanks for pointing it out.